knotx-core/src/main/java/io/knotx/options/SnippetOptions.java

```diff
@@ -26,10 +29,15 @@
 public class SnippetOptions {
 
   private static final String DEFAULT_TAG_NAME = "script";
+  private static final String DEFAULT_FALLBACK_TAG_NAME = "knotx:fallback";
   private static final String DEFAULT_PARAMS_PREFIX = "data-knotx-";
+  private List<FallbackMetadata> DEFAULT_FALLBACKS = Lists.newArrayList(new FallbackMetadata("BLANK", "<knotx:fallback data-knotx-fallback-id=\"BLANK\"></knotx:fallback>"));
```

Comment:
Shouldn't the default content be:
`new FallbackMetadata("BLANK", "<"+tagName+":fallback "+paramsPrefix+"fallback-id=\"BLANK\"></"+tagName+":fallback>")`
?

Also, cosmetics: please fix the name of this field - it is not declared as a constant but has uppercase name.

Reply:
Method adjusted